### PR TITLE
Real SSR for ads

### PR DESCRIPTION
**Pull request by DAreRodz**
Closes https://github.com/frontity/frontity/issues/69

### CODE REVIEW

File: core/packages/ads/shared/components/Ad/AdSense/index.js [hunk 1]
```diff
@@ -1,7 +1,12 @@
+/* eslint-disable react/no-danger */
 import React, { PureComponent, Fragment } from 'react';
 import PropTypes from 'prop-types';
 import styled from 'styled-components';
 import { Helmet } from 'react-helmet';
+import Script from '../../Script';
+
+// eslint-disable-next-line
+import call from 'raw-loader!babel-loader?forceEnv=devClient!./functions/call';
```
Comment (orballo) [inline core/packages/ads/shared/components/Ad/AdSense/index.js]: This loader is obsolete. Once https://github.com/frontity/frontity/pull/124 is merged, `dev` should be merged to this branch to fix bugs.

**Review by orballo — CHANGES_REQUESTED** [review]

File: core/packages/ads/shared/components/Ad/DoubleClick/index.js [hunk 2]
```diff
@@ -2,8 +2,11 @@ import React, { PureComponent, Fragment } from 'react';
 import PropTypes from 'prop-types';
 import styled from 'styled-components';
 import { Helmet } from 'react-helmet';
+import Script from '../../Script';
+
+// eslint-disable-next-line
+import call from 'raw-loader!babel-loader?forceEnv=devClient!./functions/call';
```
Comment (orballo) [inline core/packages/ads/shared/components/Ad/DoubleClick/index.js]: This loader is also obsolete. https://github.com/frontity/frontity/pull/118/files/2839af4f9caeaa6936375cea3ed816d1e4b1864c#r225954813

**Comment by luisherranz** [comment]
This is interesting. Let me know how you solve it guys.

**Comment by DAreRodz** [comment]
@orballo ready to be reviewed again!

**Review by orballo — APPROVED** [review]

**Pull request merged**